test/jdk/java/awt/TitledBorder/TitledBorderTest.java

```diff
@@ -0,0 +1,97 @@
+import java.awt.BorderLayout;
+import java.awt.Graphics2D;
+import java.awt.Robot;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+import javax.imageio.ImageIO;
+import javax.swing.BorderFactory;
+import javax.swing.JCheckBox;
+import javax.swing.JFrame;
+import javax.swing.JPanel;
+import javax.swing.UIManager;
+import javax.swing.UIManager.LookAndFeelInfo;
+import javax.swing.UnsupportedLookAndFeelException;
+import javax.swing.SwingUtilities;
+
+/*
+ * @test
+ * @key headful
+ * @bug 8279614
+ * @summary The left line of the TitledBorder is not painted on 150 scale factor
+ * @requires (os.family == "windows")
+ * @run main TitledBorderTest
+ */
+public class TitledBorderTest {
+
+  public static JFrame frame;
+  public static JPanel parentPanel;
+  public static JPanel childPanel;
+
+  public static void main(String[] args) throws Exception {
+    LookAndFeelInfo laf = UIManager.getInstalledLookAndFeels()[3];
+    System.out.println(laf);
+    SwingUtilities.invokeAndWait(() -> setLookAndFeel(laf));
+    SwingUtilities.invokeAndWait(() -> createAndShowGUI());
+
+    Robot robot = new Robot();
+
+    BufferedImage buff = new BufferedImage(frame.getWidth()*2, frame.getHeight()*2,
+            BufferedImage.TYPE_INT_ARGB);
+    Graphics2D graph = buff.createGraphics();
+    graph.scale(1.5, 1.5);
+    frame.paint(graph);
+    graph.dispose();
+
+    robot.waitForIdle();
+    if (buff.getRGB(20,80) != -6250336) {
+      System.out.println("Color " + buff.getRGB(21, 80));
+      saveImage(buff, "test.png");
+      throw new RuntimeException("Border was clipped or overdrawn.");
+    }

```

Comment:
Can we draw directly to the buffered image using the scaled graphics to reproduce the bug?

Reply:
Yea, the test fails before the fix, and when inspecting the saved image it looks identical to the actual frame.